test/integration/addons_test.go

```diff
@@ -66,6 +67,15 @@ func TestAddons(t *testing.T) {
 			t.Fatalf("Failed setting GOOGLE_CLOUD_PROJECT env var: %v", err)
 		}
 
+		// MOCK_GOOGLE_TOKEN forces the gcp-auth webhook to use a mock token instead of trying to get a valid one from the credentials.
+		err = os.Setenv("MOCK_GOOGLE_TOKEN", "true")
```

Comment:
Can we use `t.Setenv` here instead? Then we don't have to do the cleanup ourselves.

Reply:
Done